src/utils.ts

```diff
@@ -23,6 +23,7 @@ export const OMITTING_KEYS_REGEXES = [
   'Signature',
   'Credential',
   'Authorization',
+  'authorization',
```

Comment:
There is an ignorecase, why did you add that?
![image](https://user-images.githubusercontent.com/47108628/139585708-c9652cac-013f-4e78-b2ce-05aae8367e0f.png)